pkg/api/helpers.go

```diff
@@ -118,6 +118,12 @@ func NewDeleteOptions(grace int64) *DeleteOptions {
 	return &DeleteOptions{GracePeriodSeconds: &grace}
 }
 
+// NewListOptions returns a ListOptions indicating the resource should be
+// listed with at least given 'resourceVersion'.
+func NewListOptions(resourceVersion string) *ListOptions {
```

Comment:
I never thought I'd argue against a constructor, but is a constructor for list options really useful? I'd expect it's better to just have people set the fields they care about. That way if you add a field you don't need to change the constructor and all invocations.

(caveat: the above doesn't apply if there's more to be done than just setting fields)